packages/stellar-axelar-std/src/interfaces/operatable.rs

```diff
@@ -18,10 +18,7 @@ pub trait OperatableInterface {
 
 /// Default implementation of the [OperatableInterface] trait.
 pub fn operator(env: &Env) -> Address {
-    env.storage()
-        .instance()
-        .get(&storage::operator::DataKey::Interfaces_Operator)
-        .expect("operator must be set during contract construction")
+    storage::operator::try_interfaces_operator(env).expect("operator must be set during contract construction")
```

Comment:
```suggestion
    storage::operator::interfaces_operator(env)
```
this will panic for you